spec.html

```diff
@@ -7652,19 +7650,44 @@ <h1>Jobs and Host Operations to Enqueue Jobs</h1>
     </ul>
 
     <emu-note>
-      <p>Host environments are not required to treat Jobs uniformly with respect to scheduling. For example, web browsers and Node.js treat Promise-handling Jobs as a higher priority than other work; future features may add Jobs that are not treated at such a high priority.</p>
+      Host environments are not required to treat Jobs uniformly with respect to scheduling. For example, web browsers and Node.js treat Promise-handling Jobs as a higher priority than other work; future features may add Jobs that are not treated at such a high priority.
     </emu-note>
 
+    <p>An invocation of a Job <dfn id="job-propagatesactivescriptormodule">propagates the script or module</dfn> _scriptOrModule_ (a Script Record, a Module Record, or *null*) if all of the following conditions are true:</p>
+    <ul>
+      <li>GetActiveScriptOrModule() is _scriptOrModule_.</li>
```

Comment:
It would help to say at which point these two properties are expected to be true - before the Job executes? After it? Something else? It is not the case that this must hold for the duration of the invocation, I believe.

Reply:
Will clarify.

Reply:
Ended up rephrasing this property as applying on executions at a particular point in time as well.